flask-server/routes/content.py

```diff
@@ -2,41 +2,40 @@
 from exts import db
 from models.content_models import Books, Anime, Games
 
-
 content_ns = Namespace("content", description="A namespace for content recommendations.")
 
-books_model=content_ns.model("Books", {
-    "Book_ID": fields.Integer,
-    "Book_Name": fields.String,
-    "Book_Author": fields.String,
-    "Book_Genre": fields.String,
-    "Price": fields.Float,
-    "Book_Script": fields.String,
-    "Book_Image": fields.String
+books_model = content_ns.model("Books", {
+    "book_id": fields.Integer,
+    "book_name": fields.String,
+    "book_author": fields.String,
+    "book_genre": fields.String,
+    "price": fields.Float(description="RRP"),
+    "book_script": fields.String(description="Description of the book"),
+    "book_image": fields.String
 })
 
-anime_model=content_ns.model("Anime", {
-    "Anime_ID": fields.Integer,
-    "Anime_Name": fields.String,
-    "Anime_Genre": fields.String,
-    "Where_TW": fields.String,
-    "Anime_Script": fields.String,
-    "Anime_Image": fields.String
+anime_model = content_ns.model("Anime", {
+    "anime_id": fields.Integer,
+    "anime_name": fields.String,
+    "anime_genre": fields.String,
+    "where_tw": fields.String(description="Where to watch the anime"),
+    "anime_script": fields.String(description="Description of the anime"),
+    "anime_image": fields.String
 })
 
-games_model=content_ns.model("Games", {
-    "Game_ID": fields.Integer,
-    "Game_Name": fields.String,
-    "Game_Genre": fields.String,
-    "W_Console": fields.String,
-    "Price": fields.Float,
-    "Game_Script": fields.String,
-    "Game_Image": fields.String
+games_model = content_ns.model("Games", {
+    "game_id": fields.Integer,
+    "game_name": fields.String,
+    "game_genre": fields.String,
+    "w_console": fields.String(description="Which consoles the game is available on"),
+    "price": fields.Float(description="RRP"),
+    "game_script": fields.String(description="Description of the game"),
+    "game_image": fields.String
 })
 
 
 @content_ns.route("/books")
```

Comment:
For this file it might be interesting to look into query parameters. 
These will definitely work, but it might be interesting to search for Thrillers of a certain author as well.

Reply:
Will look into mulitple query parameters in the future, that is a nice idea!